app/views/v2/correspondence_case_heading.scala.html

```diff
@@ -0,0 +1,49 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.viewmodels.CaseHeaderViewModel
+
+@this(
+        caseStatus: views.html.v2.case_status
+    )
+
+@(
+caseHeader: CaseHeaderViewModel,
+headingText: Option[String] = None,
+headingPrefix: Option[String] = None
+)(implicit messages: Messages)
+
+<div id="case-heading" class="case-heading mt-5 mb-4">
+
+    <div id="case-reference" class="case-reference">
+        @messages("case.heading.secondary.nosuffix", "Correspondence", caseHeader.referenceNumber)
+    </div>
+
+    <h1 class="heading-xlarge case-title">
+        @headingText.map(HtmlFormat.escape(_)).getOrElse {
+        @headingPrefix.fold(HtmlFormat.escape(caseHeader.goodsName)) { prefix =>
+        @messages("case.title", prefix, caseHeader.goodsName)
+        }
+        }
+    </h1>
+
+    <div>
+        <h3 class="govuk-heading-s">@caseHeader.businessName.getOrElse("")</h3>
+        <h3 class="govuk-heading-s">@caseHeader.contactEmail.getOrElse("")</h3>
+    </div>
+
+    @caseStatus(caseHeader.caseStatus)
+</div>
```

Comment:
Does this need a new template? I think we can just reuse the existing `case_heading`

Reply:
(I think @warrenskiwork1979 has done something similar to this on DIT-2536)

Reply:
We created this heading because on the prototype the heading for the correspondence and misc types cases are slightly different then what we have for atars and liabilities. e.g it display the email of the contact